src/clear-all-refinements/index.ts

```diff
@@ -0,0 +1,19 @@
+import { alias, tag, Tag } from '@storefront/core';
+
+@alias('clearAllRefinements')
+@tag('gb-clear-all-refinements', require('./index.html'))
+class ClearAllRefinements {
+
+  state: ClearAllRefinements.State = {
+    onClick: () => this.actions.resetRefinements()
```

Comment:
instead of putting `onClick()` in the `state`, just make it an instance method and access it in the template with `on-click="{ onClick }"`